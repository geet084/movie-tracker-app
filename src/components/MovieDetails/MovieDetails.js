import React from 'react'
import { Link } from 'react-router-dom'

export const MovieDetails = (props) => {
  console.log(props)
  return (
    <div>
      <Link to='/'>
      <h4>All Movies</h4>
      </Link>
      <img src={`http://image.tmdb.org/t/p/w500/${props.backdrop_path}`} alt=""/>
      <h4>{props.title}</h4>
      <h4>{props.overview}</h4>
      <h4>{props.release_date}</h4>
      <h4>{props.vote_average}</h4>
    </div>
  )
}

export default MovieDetails;