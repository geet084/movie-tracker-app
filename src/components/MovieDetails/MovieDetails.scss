.movie-details-div {
  background-color: white;
  position: fixed;
  margin: 10px auto;
  width: 1000px;
  height: 563px;
  border: 1px solid black;
  z-index: 12;
  left: 50%;
  top: 50%; 
  margin-left: -500px;
  margin-top: -250px;
  position: fixed;
  box-shadow: 2px 2px 8px black;
}

.movie-backdrop {
  position: absolute;
  width: 998px;
  z-index: 13;
}

.return-link {
  text-shadow: 2px 2px 8px black;
  color: white;
  margin: 20px;
  position: absolute;
  z-index: 14;
}

.movie-text-div {
  height: 275px;
  position: relative;
  top: 286px;
  background-color: rgba(0, 0, 0, 0.6);
  z-index: 14;
  padding: 20px;
}

.movie-details-div {
  z-index: 15;
}

.movie-details {
  color: white;
}

#title-text {
  margin-top: 5px;
}

#overview-text {
  text-align: justify;
  width: 700px;
}
