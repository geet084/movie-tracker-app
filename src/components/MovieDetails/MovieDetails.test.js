import React from 'react'
import { shallow } from 'enzyme'
import MovieDetails from './MovieDetails'

describe('MovieDetails', () => {
  it.skip('should', () => {
  })
})