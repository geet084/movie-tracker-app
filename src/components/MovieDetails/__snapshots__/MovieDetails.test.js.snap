// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails should match the snapshot with all data passed in 1`] = `undefined`;
